app/models/search-result.ts

```diff
@@ -90,18 +90,35 @@ export default class SearchResultModel extends Model {
     get affiliatedEntities() {
         if (this.resourceType === 'user') {
             if (this.resourceMetadata.affiliation) {
-                return this.resourceMetadata.affiliation.map((item: any) =>
-                    ({ name: getSingleOsfmapValue(item, ['name']), absoluteUrl: item['@id'] }));
+                return this.resourceMetadata.affiliation.map((item: any) => ({
+                    name: getSingleOsfmapValue(item, ['name']),
+                    absoluteUrl: item['@id'],
+                }));
             }
         } else if (this.resourceMetadata.creator) {
-            return this.resourceMetadata.creator?.map((item: any) =>
-                ({ name: getSingleOsfmapValue(item, ['name']), absoluteUrl: item['@id'] }));
+            return this.getSortedContributors(this.resourceMetadata);
         } else if (this.isContainedBy?.[0]?.creator) {
-            return this.isContainedBy[0].creator.map((item: any) =>
-                ({ name: getSingleOsfmapValue(item, ['name']), absoluteUrl: item['@id'] }));
+            return this.getSortedContributors(this.isContainedBy?.[0]);
         }
     }
 
+    private getSortedContributors(base) {
+        const objectOrder = Object.fromEntries(
+            base.qualifiedAttribution?.map((item: any) => [
+                getSingleOsfmapValue(item, ['agent']),
+                Number.parseInt(
+                    getSingleOsfmapValue(item, ['osf:order']),
```

Comment:
sidenote: hmm, whoops, integers should be json-serialized as integers, not `{"@value": "0", "@type": "xsd:integer"}`... but this works and should continue working if `getSingleOsfmapValue` returns an integer... so 👍